[IVANCHUK] Bump version of react-ui-components to 0.11.37

mzazrivec opened the pull request:
PRs from 0.11.27:
https://github.com/ManageIQ/react-ui-components/pull/129
https://github.com/ManageIQ/react-ui-components/pull/134
https://github.com/ManageIQ/react-ui-components/pull/135
https://github.com/ManageIQ/react-ui-components/pull/133
https://github.com/ManageIQ/react-ui-components/pull/119
https://github.com/ManageIQ/react-ui-components/pull/131
https://github.com/ManageIQ/react-ui-components/pull/136
https://github.com/ManageIQ/react-ui-components/pull/137
https://github.com/ManageIQ/react-ui-components/pull/139
https://github.com/ManageIQ/react-ui-components/pull/140

master PR: https://github.com/ManageIQ/manageiq-ui-classic/pull/5979

simaishi commented:
@mzazrivec Updating packages in Ivanchuk requires yarn.lock to be updated together. Also, 'master' branch currently requires `~0.11.35`, so it's a bit strange Ivanchk requires newer version than master.

cc @himdel

himdel commented:
This definitely needs the `yarn.lock` change, agreed :)

As for 0.11.35 vs 0.11.36, that's fine, the difference between 35 and 36 is just a gettext fix (https://github.com/ManageIQ/react-ui-components/pull/139),
**but** we probably want 0.11.37 which also has the catalog updated (https://github.com/ManageIQ/react-ui-components/pull/140), right @mzazrivec ?

mzazrivec commented:
@himdel I could probably just create a PR for master and have it cherry-picked into ivanchuk, right?

himdel commented:
Well, the package.json change means ivanchuck needs a yarn.lock update too. But we gitignore yarn.lock on master.

So, yes, you could just create a master PR, get that backported and we could create a separate update yarn lock on ivanchuk PR. But I think it does need an ivanchuk PR in some form.

himdel commented:
@mzazrivec Feel frere to create a master, ivanchuk/yes PR, I'll be updating yarn.lock on monday.

mzazrivec commented:
@simaishi @himdel 
* created https://github.com/ManageIQ/manageiq-ui-classic/pull/5979 for master
* updated `yarn.lock` in this PR
* added list of PRs that `react-ui-components` version `0.11.37` brings (since `0.11.27`)
